todo_app/tests/config.py

```diff
@@ -0,0 +1,8 @@
+# Trello configuration
+TRELLO_BOARD_ID='64ae6f476f946f8c451a5cb2'
+TRELLO_APIKEY='7d069c46c188afd7a7eaeb18851b9fac'
+TRELLO_TOKEN='ATTAcff8cd91ef9d8108a296d4954de006c36a5f27507477246e8cd87d76edbdf881F66B1476' #actually gen a proper one using instructions!
+#TRELLO_SECRET='1606460f48bdbc6a99ada2dcff7b84fcb767607168a3a5eb274c1d356677dc09'
```

Comment:
Secrets leaked!

We always want to avoid this, and the moment they're pushed up to GitHub we should treat these as compromised, even if this had been a private repository. There are ways to carefully remove them from the Git tree, but personally I lean towards not trusting that and rotating these - so e.g. [revoke your token here](https://developer.atlassian.com/cloud/trello/guides/rest-api/authorization/#revoking-tokens) and [generate a new one](https://trello.com/app-key) so that we can just accept that they were leaked but are of no value now

For our tests, we want these to hold fake credentials, and I'm a big fan of _obviously_ fake credentials so that future developers don't panic, nor do they miss if real ones accidentally get added:
```suggestion
TRELLO_APIKEY=this-is-a-fake-key
TRELLO_TOKEN=this-is-a-fake-token
```